### PR TITLE
i18n(es): update getting started

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Update `getting-started.md` #1126

### CODE REVIEW

File: src/pages/es/getting-started.md [hunk 1]
```diff
@@ -2,85 +2,89 @@
...
+i18nReady: true
 ---
-Generador de sitios estáticos  🚀  Trae tu propio Framework  🚀  Utiliza menos JavaScript
 
-:::tip
-Tienes un proyecto desactualizado? Sigue la [guía de migración](/es/migrate/) para ponerte al día con la v1.0 beta!
-:::
+#### ¿Qué es Astro?
+
+Astro es un **web framework** **todo incluido** para crear páginas web **rápidas** y **centradas en el contenido**.
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:14]: Alternative, not a strong opinion about this one :P

```suggestion
Astro es un **framework web con todo incluido** para crear páginas web **rápidas** y **centradas en el contenido**.
```

File: src/pages/es/getting-started.md [hunk 2]
```diff
@@ -2,85 +2,89 @@
...
 
+#### Características clave
 
-## Prueba Astro
+- **Islas de componentes:** Una nueva arquitectura web para crear sitios web más rápidos.
+- **Diseño de API que prioriza el servidor:** Elimina la costosa hidratación de los dispositivos de tus usuarios.
+- **Cero JS, de forma predeterminada:** Sin sobrecarga del entorno de ejecución de JavaScript que lo ralentice.
+- **Listo para Edge:** Despliegue en cualquier lugar, incluso en un entorno de ejecución global Edge como Deno o Cloudflare.
+- **Personalizable:** Tailwind, MDX y más de 100 integraciones entre las que elegir.
+- **UI-Agnostico:** Compatible con React, Preact, Svelte, Vue, Solid, Lit y más.
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:23]:
```suggestion
- **UI-Agnóstico:** Compatible con React, Preact, Svelte, Vue, Solid, Lit y más.
```

File: src/pages/es/getting-started.md [hunk 3]
```diff
@@ -2,85 +2,89 @@
...
+Generador de sitios estáticos  🚀  Trae tu propio Framework  🚀  Utiliza menos JavaScript
 
-Empieza con Astro de manera sencilla, puedes hacerlo directo en tu navegador o máquina local!
+<!-- - Cargado de componente **`client:visible`:** Si tu usuario nunca lo ve, nunca se carga. -->
+<!-- - **Optimizaciones de imagen:** Nuestro propio componente `<Image />` -->
+<!-- - **Compatibilidad con TypeScript** -->
+<!-- - **Enrutamiento basado en archivos:** Cada archivo en el directorio de páginas se convierte en una ruta. -->
 
-### Playground en línea
+Consulte nuestro resumen detallado de [¿Por qué Astro?](/es/concepts/why-astro/) para obtener más información sobre que hace a Astro especial. ✨
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:31]:
```suggestion
Consulta nuestro resumen detallado de [¿Por qué Astro?](/es/concepts/why-astro/) para obtener más información sobre qué hace a Astro especial. ✨
```

File: src/pages/es/getting-started.md [hunk 4]
```diff
@@ -2,85 +2,89 @@
...
 
-Visita [astro.new](https://astro.new) para "ver antes de comprar". Puedes elegir de entre una gran variedad de plantillas para construir lo que quieras teniendo como base una versión completa y funcional de Astro directamente en tu navegador!
+## ¡Prueba Astro en tu navegador!
+
+Visite [astro.new](https://astro.new/) y elija entre una variedad de plantillas para comenzar. ¡Juega con una versión completa y funcional de Astro directamente en tu navegador!
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:35]:
```suggestion
Visita [astro.new](https://astro.new/) y elige entre una variedad de plantillas para comenzar. ¡Juega con una versión completa y funcional de Astro directamente en tu navegador!
```

File: src/pages/es/getting-started.md [hunk 5]
```diff
@@ -2,85 +2,89 @@
...
-
-🏗️ Crea tu primer [layout](/es/core-concepts/layouts/).
-
-🏗️ Agrega [CSS y estilos](/es/guides/styling/) a tu proyecto. 
-
-*... encuentra aún más en la categoría **Características***
-
+## Empieza tu primer proyecto
+
+Crea un nuevo proyecto de Astro localmente con nuestro asistente de CLI `create-astro`!
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:46]:
```suggestion
¡Crea un nuevo proyecto de Astro localmente con nuestro asistente de CLI `create-astro`!
```

File: src/pages/es/getting-started.md [hunk 6]
```diff
@@ -2,85 +2,89 @@
...
+</PackageManagerTabs>
+
+Nuestra [guía de instalación](/es/install/auto/) tiene instrucciones paso a paso para instalar Astro utilizando tu gestor de paquetes favorito.
 
 ## Aprende Astro
 
-Encuentra ejemplos de algunos de los patrones y conceptos clave de un sitio de Astro!
-
-📚 Lee más acerca de la [estructura de proyecto](/es/core-concepts/project-structure/) de Astro.
+¡Encuentre ejemplos de los conceptos y patrones claves de un proyecto de Astro!
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:73]: Me suena más natural "conceptos clave"

```suggestion
¡Encuentra ejemplos de los patrones y conceptos clave de un proyecto de Astro!
```

File: src/pages/es/getting-started.md [hunk 7]
```diff
@@ -2,85 +2,89 @@
...
 
-📚 Aprende sobre las [directivas en plantillas](/es/reference/directives-reference/) de Astro.
+📚 [Agrega tu primera página](/es/core-concepts/astro-pages/) a tu proyecto.
 
-📚 Explora la [API de runtime](/es/reference/api-reference/) de Astro.
+📚 Lea más sobre la [estructura del proyecto](/es/core-concepts/project-structure/) de Astro.
 
-*... encuentra más material en la pestaña **Referencia***
+📚 Aprenda sobre el [enrutamiento basado en archivos](/es/core-concepts/routing/) de Astro.
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:79]:
```suggestion
📚 Aprende sobre el [enrutamiento basado en archivos](/es/core-concepts/routing/) de Astro.
```

File: src/pages/es/getting-started.md [hunk 8]
```diff
@@ -2,85 +2,89 @@
...
-Encuentra ejemplos de algunos de los patrones y conceptos clave de un sitio de Astro!
-
-📚 Lee más acerca de la [estructura de proyecto](/es/core-concepts/project-structure/) de Astro.
+¡Encuentre ejemplos de los conceptos y patrones claves de un proyecto de Astro!
 
-📚 Aprende sobre las [directivas en plantillas](/es/reference/directives-reference/) de Astro.
+📚 [Agrega tu primera página](/es/core-concepts/astro-pages/) a tu proyecto.
 
-📚 Explora la [API de runtime](/es/reference/api-reference/) de Astro.
+📚 Lea más sobre la [estructura del proyecto](/es/core-concepts/project-structure/) de Astro.
```
Comment (agustinmulet) [inline src/pages/es/getting-started.md:77]:
```suggestion
📚 Lee más sobre la [estructura del proyecto](/es/core-concepts/project-structure/) de Astro.
```

**Review by agustinmulet — COMMENTED** [review]
Reviewed! Good job :)

**Comment by Yan-Thomas** [comment]
So many merges today! 🤩 Thank you @kevinzunigacuellar, @agustinmulet for this amazing work!

**Pull request merged**